pulp_2to3_migration/app/plugin/content.py

```diff
@@ -372,9 +372,11 @@ def get_remote_by_importer_id(importer_id):
                         continue
                     if remotes:
                         for remote in remotes:
+                            # ensure trailing slash
+                            url = remote.url.rstrip("/") + "/"
```

Comment:
this theoretically can be a mirrorlist, but at this point we have no way to distinguish it from a normal single feed.

Reply:
i have added a trailing slash check here because pulp3 remote don't have any sort of validation on the url https://pulp.plan.io/issues/8340

Reply:
I think the reason is mirror list and we might want webserver to handle it properly. So I'm not sure if we want to enforce slash at the end.

Reply:
we should, I think,  because the webserver won't help here since the DA artifact will have a wrong url.
I can remove it but if there has been provided a feed without trailing slash we will have this url wrong.
On the other hand if there has been provided a mirrorlist adding a slash will also construct a wrong url.

Reply:
I'm not suggesting to remove it here, I'm questioning whether we want to enforce it in rpm plugin. +1 to keep it here.

Reply:
ahh, no-no i answered myself on that ticket that we cannot actually enforce it in the rpm_plugin